### PR TITLE
Updated the cancel ritual gizmo for 1.3.3101

### PR DESCRIPTION
**Pull request by SokyranTheDragon**

### CODE REVIEW

File: Source/Client/Sync/Game/SyncDelegates.cs [hunk 1]
```diff
@@ -217,6 +217,25 @@ static void ReopenTradingWindowLocally(Caravan caravan, Command __result)
                 original();
             };
         }
+
+        [MpTranspiler(typeof(LordJob_Ritual), nameof(LordJob_Ritual.GetCancelGizmo), 0)]
```
Comment (Zetrith) [inline Source/Client/Sync/Game/SyncDelegates.cs:221]: Why is this necessary? Why not a normal SyncMethod on LordJob_Ritual.Cancel?

Comment (SokyranTheDragon) [reply Source/Client/Sync/Game/SyncDelegates.cs:221]: I didn't want to do sync the Cancel method itself as it could be called in places where syncing it is not needed, especially considering it's a public method.

Comment (Zetrith) [reply Source/Client/Sync/Game/SyncDelegates.cs:221]: The method is currently not called in any undesirable places and it's so specific that that's unlikely to change + using a SyncMethod would remove the need for a transpiler. I think you should change it.
